### ISSUE TITLE
Language Option for Food Search

### ISSUE THREAD

**Issue by MasterOfTheTiger** [SecUSo/privacy-friendly-food-tracker#76]
When I search for food, I am greeted with options in French. I do not see any option to change the language, but I do see that the API you use does support English.

Did I miss something or is there no option to change languages from within the app?

**Comment by Kamuno** [comment]
I just checked and it seems the App is either using german or "world" for the api.
It would indeed be nice if you could choose your country or if it atleast would be chosen for you according to your device setting.

These are the supported countries:
https://world.openfoodfacts.org/data/taxonomies/countries.json

**Comment by MasterOfTheTiger** [comment]
I was pretty sure I saw French (but I don't want to embarass myself too much so I'll check again).

**Comment by Kamuno** [comment]
- Add option to search in world database if no results in local database.

**Comment by christianfl** [comment]
> 
> 
>     * Add option to search in world database if no results in local database.
> 
>     * Add option in settings to choose country

There's unfortunately another case: The country specific search can find some product(s) but not all so that the world search shows more options.

**Comment by anddam** [comment]
> Edit: Wouldn't it be better to just use "world" until any manual exclusive country setting is there? This would be very easy to change just within _/app/src/main/java/org/secuso/privacyfriendlyfoodtracker/network/ApiManager.java_

Yes it would. I spoke about this on OFF slack channel a few months ago and got confirmation.

I am surprised this is still an open issue, this makes the application **unusable** for those affected by the bug.
